typescript-json-schema.ts

```diff
@@ -9,7 +9,7 @@ export { Program, CompilerOptions, Symbol } from "typescript";
 
 const vm = require("vm");
 
-const REGEX_FILE_NAME_OR_SPACE = /(\bimport\(".*?"\)|".*?")\.| /g;
+const REGEX_FILE_NAME_OR_SPACE = /(\bimport\(".*?"(, \{ assert: \{ "resolution-mode": "(import|require)" \} \})?\)|".*?")\.| /g;
```

Comment:
Hi @FoxxMD,
thank you for pointing in the right direction. I had to use a different regex when using `typescript@22.14.0` and `typescript@5.8.3` with `esm` modules:

```suggestion
const REGEX_FILE_NAME_OR_SPACE = /(\bimport\(.*?\)|".*?")\.| /g;
```
I'm patching this line with the help of [patch-package](https://www.npmjs.com/package/patch-package) into `typescript-json-schema` to make it working again.

@YousefED Thank you for this great package! I would like to politely ask if this regular expression could be updated to work with ESM packages.